gio/src/socket.rs

```diff
@@ -61,6 +68,21 @@ impl AsRawSocket for Socket {
     }
 }
 
+#[doc(alias = "GInputMessage")]
+pub struct InputMessage {
```

Comment:
These two can probably be made `#[repr(transparent)]` wrappers around the C types